e2e/__tests__/globals.test.ts

```diff
@@ -106,11 +144,11 @@ test('only', () => {
 
   writeFiles(TEST_DIR, {[filename]: content});
   const {stderr, exitCode} = runJest(DIR);
-  expect(exitCode).toBe(0);
 
   const {summary, rest} = extractSummary(stderr);
   expect(wrap(rest)).toMatchSnapshot();
   expect(wrap(summary)).toMatchSnapshot();
+  expect(exitCode).toBe(0);
```

Comment:
Nicer to see the snapshot diff rather than just the wrong number in case something goes wrong, so I changed this for all test cases in here

Reply:
yeah, good call. I've done that before as well for the same reason